src/api/plugins/video/warren_video/api.py

```diff
@@ -43,20 +44,22 @@ async def views(
     indicator = klass_mapping.get((complete, unique), DailyViews)(
         video_id=video_id, span_range=DatetimeRange.parse_obj(filters)
     )
-    logger.debug("Will compute indicator %s", indicator)
+    logger.debug(f"Will compute indicator {type(indicator).__name__}")
+    logger.debug(f"From {indicator.span_range.since} to {indicator.span_range.until}")
```

Comment:
Pylint would have told you that we are not supposed to use f-strings in logs, but plain old-style logs formatting:

```suggestion
    logger.debug("Will compute indicator %s", type(indicator).__name__)
    logger.debug("From %s to %s", indicator.span_range.since, indicator.span_range.until)
```

References:

1. https://pylint.pycqa.org/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html
2. https://www.reddit.com/r/learnpython/comments/xx14jw/why_not_use_fstrings_in_logging/

Reply:
The current linter triggers no warning. Alright, thanks for the references, kinda suck that we still need to use this old syntax.

Reply:
fixed !